elements/pfe-navigation/src/pfe-navigation-item.scss

```diff
@@ -28,10 +30,13 @@ $variables: (
   line-height: 1.2;
   
```

Comment:
I'm seeing an issue w/ the tray padding on mobile because we are missing accordion variables in the nav.
![image](https://user-images.githubusercontent.com/456863/71270806-7958a680-2320-11ea-903f-5d9ae5e421dd.png)

So this works, but is obviously a mess of variables. 

```css
  --pfe-accordion--PaddingBase: var(--pfe-theme--container-spacer, 16px);
  --pfe-accordion--PanelContainer--padding: 0 calc(var(--pfe-accordion--PaddingBase) * 3) var(--pfe-accordion--PaddingBase) calc(var(--pfe-accordion--PaddingBase) * 1.5);
  --pfe-navigation--PanelContainer--padding: var(--pfe-accordion--PanelContainer--padding);  // this the the necessary one, we could change up the value though
```

I tried doing this, which seems more logical but it didn't work
```css
  --pfe-navigation--PanelContainer--padding: var(--pfe-navigation--Trigger--padding);
```

Reply:
Thanks for catching this @starryeyez024 
I added these variables to `pfe-navigation.scss`